backend/Builtins.icl

```diff
@@ -463,6 +464,16 @@ bs_module = (["module", "definition", "implementation", "system", "definition mo
 		]
 	})
 
+bs_pattern_named = (["=:"],
+	{ syntax_title         = "newtype definition (experimental)"
+	, syntax_code          = [":: ... =: T c"]
+	, syntax_description   = "Define a newtype, a newtype is a type synonym that is treated as a real type."
+	, syntax_doc_locations = []
+	, syntax_examples      =
+		[ EX  "Function"         ":: T = T Int"
```

Comment:
- The extra spaces are not needed here.
- The `=` should be `=:`.
- You could also add an example with a type variable, `:: T a =: T a`.
- You probably need to use `EX "TypeDef"` instead of `"Function"` to get the proper highlighting.